users/urls.py

```diff
@@ -1,6 +1,12 @@
 from django.urls import path
 from . import views
+from .views import login_root, login_success, login_fail
 
+from django.conf.urls import url
 urlpatterns = [
     path('', views.index, name='index'),
+    url(r'^login/$', login_root, name='login_root'),
+    url(r'^login/success/$', login_success, name='login_success'),
```

Comment:
this page should not be accessible through an http request maybe replace it with code we discussed earlier